nav2_costmap_2d/test/integration/obstacle_tests.cpp

```diff
@@ -405,5 +406,103 @@ TEST_F(TestNode, testRaytracing) {
   lethal_count = countValues(*(layers.getCostmap()), nav2_costmap_2d::LETHAL_OBSTACLE);
 
   ASSERT_EQ(lethal_count, 1);
+}
+
+/**
+ * Test dynamic parameter setting of voxel layer
+ */
+TEST_F(TestNode, testDynParamsSetVoxel)
+{
+  auto node = std::make_shared<rclcpp_lifecycle::LifecycleNode>("voxel_dyn_param_test");
```

Comment:
Again, this looks like an unnecessary code.

Reply:
Done!